store/src/java-test/com/zimbra/cs/datasource/imap/RemoteFolderTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2011, 2013, 2014, 2016, 2022 Synacor, Inc.
```

Comment:
for copyright just keep the year 2022 and remove the others. 
Since this test class is added newly and wasn't present earlier, will just keep the current year and the files which are getting modified there if the current year is not present there we add the current year to existing list.

Reply:
Thanks , wasn't aware of this fixed the Year